core/controllers/classifier.py

```diff
@@ -14,16 +14,19 @@
 
 """Controllers for communicating with the VM for training classifiers."""
 
+import datetime
 import hashlib
 import hmac
 import json
 
 from core.controllers import base
 from core.domain import classifier_services
 from core.domain import config_domain
+from core.domain import exp_services
 
 import feconf
 
+TTL = 5*60*60
```

Comment:
As per our last discussion @anmolshkl this has to be reviewed. I'll update this in feconf after the review.